nannyml/drift/univariate/calculator.py

```diff
@@ -76,7 +80,12 @@ def __init__(
         ...    res.plot(kind='drift', column_name=column_name, method=method).show()
         """
         super(UnivariateDriftCalculator, self).__init__(
-            chunk_size, chunk_number, chunk_period, chunker, timestamp_column_name
+            chunk_size,
+            chunk_number,
+            chunk_period,
+            chunker,
+            timestamp_column_name,
+            calculation_method,
```

Comment:
Cfr. my comment on the base class: don't have the `calculation` in the `super` call here, but just add it as an instance variable here.

Reply:
resolved, please take a look @nnansters

Reply:
Looks good!